src/mbgl/style/source_impl.cpp

```diff
@@ -168,7 +172,8 @@ void Source::Impl::updateTiles(const UpdateParameters& parameters) {
                                    parameters.debugOptions & MapDebugOptions::Collision };
 
     for (auto& pair : tiles) {
-        pair.second->setPlacementConfig(config);
+        auto& tile = pair.second;
+        tile->setPlacementConfig(config);
```

Comment:
I prefer the original -- `pair.second->setPlacementConfig(config);` -- over introducing any temporary variables. It's clear from the previous line that you're iterating over tiles.